Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Build the manager binary
-FROM golang:1.16 as builder
+FROM golang:1.17 as builder
```

Comment:
golang 1.17 is required to build the operator using v0.11.0 of the controller runtime.

Reply:
Thanks ok, there is no go version requirement for now.